e-learning-webapps/src/main/webapp/vue-app/components/ElearningDashboard.vue

```diff
@@ -97,18 +144,28 @@ export default {
         .then((data) => (this.tutoList = data))
         .catch((e) => this.errors.push(e));
     },
-    deleteTuto(id) {
-      return fetch(`/portal/rest/tuto/deleteTuto/${id}`, {
+    prepareDelete(id){
+      this.deleteId=id;
+      this.confirmDialog=true;
+    },
+    deleteTuto() {
```

Comment:
Please add all fetch methods in a dedicated js file (elearningApi.js) example: https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/common/js/SpaceService.js

And :

https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/common/main.js#L30